modules/ROOT/pages/resource-management.adoc

```diff
@@ -1,17 +1,21 @@
 = Resources management
-:description: This page defines the term "Resources" and explains what a user with the _Administrator_ profile in Bonita Portal can see and do about resources.
+:description: This page defines the term "Resources" and explains what a user can see and do about resources in the Bonita Administrator Application.
 
-This page defines the term "Resources" and explains what a user with the _Administrator_ profile in Bonita Portal can see and do about resources.
+{description}
 
 _Administrators_ can view a list of the resources filtered by type, install new resources, update existing resources, or delete resources.
 
 Here is a view of the page:
-image:images/UI2021.1/admin_resources_portal.png[Administrator Resources]
+image:images/UI2021.1/admin-application-resources-list.png[Administrator Resources]
 // {.img-responsive}
 
 == Resource definition
 
-A resource is deployed in Bonita Portal to create the User Interface of a Living Application: it can be a page, a layout, a theme, a REST API extension, or a form that will be used by several processes so it is better to store it at platform level than within the process .bar file.
+A resource is deployed in the Bonita Runtime to be used in some way by the Living Application:
```

Comment:
to be used in some way by the Living Application: => to be used in by the Living Application:

Reply:
"to be used by the Living Application:" ?